docs/source/tutorial/commercial_paper.md

```diff
@@ -775,56 +712,50 @@ information to her wallet:
 done
 ```
 
-Isabella can store multiple identities in her wallet, though in our example, she
-only uses one -- `User1@org.example.com`. This identity is currently associated
-with the basic network, rather than a more realistic PaperNet configuration --
-we'll update this tutorial soon.
-
 `addToWallet.js` is a simple file-copying program which you can examine at your
-leisure. It moves an identity from the basic network sample to Isabella's
-wallet. Let's focus on the result of this program -- the contents of
-the wallet which will be used to submit transactions to `PaperNet`:
+leisure. It moves an identity from the test network sample to Isabella's
+wallet. Let's focus on the result of this program --- the contents of
+the wallet which will be used to submit transactions to PaperNet:
 
 ```
 (isabella)$ ls ../identity/user/isabella/wallet/
 
```

Comment:
It was not clear to me if Isabella's terminal window is the Digibank window or the Magnetco corp window. We might want to clarify that.

Reply:
I changed from 
```
(magnetocorp user)$ cd commercial-paper/organization/magnetocorp/application
(magnetocorp user)$ code issue.js
```
to
```
(isabella)$ cd commercial-paper/organization/magnetocorp/application
(isabella)$ code issue.js
```
further up the doc to help make this more clear.